info.limpet.stackedcharts.model.tests/src/info/limpet/stackedcharts/model/tests/ModelTests.java

```diff
@@ -32,6 +32,7 @@ public void testReadModel()
   {
     URI resourceURI = URI.createFileURI("testRead.stackedcharts");
     Resource resource = new ResourceSetImpl().createResource(resourceURI);
+    Assert.assertNotNull("Created resource", resource);
```

Comment:
@theanuradha - I know this isn't your code.  But, if you get chance to have a look in the next 12 hours, could you find why this resource creation is failing?  The *testRead.stackedcharts* data-file is present in the plugin, so I can't see why it fails.

thanks,
Ian